CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# dbt_shopify_holistic_reporting v0.5.0
+[PR #18](https://github.com/fivetran/dbt_shopify_holistic_reporting/pull/18) includes the following changes:
+
+## 🚨 Breaking Changes 🚨:
+
+- This package now points to the following ranges of the upstream packages:
+
+```
+- package: fivetran/shopify
+  version: [">=0.10.0", "<0.11.0"]
+- package: fivetran/klaviyo
+  version: [">=0.7.0", "<0.8.0"]
+```
```

Comment:
Can you also call out that following these changes we recommend running a `dbt run --full-refresh` to capture all the latest data within incremental models.

Reply:
updated!